### DISCUSSION TITLE
Custom CopyPlugin to copy public folder contents

### DISCUSSION THREAD

**Discussion by Vinnecent** [jaredpalmer/razzle#1673] [Q&A] [answered]
## ❓Question

Is it possible to remove the CopyPlugin from createConfigAsync.js and create my own?
Could that be done with the razzle.config.js and if so how would i go about that?

I know i can get the plugins inside midifyWebpackConfig(opts) with opts.webpackConfig.plugins but i have no idea how detect which plugin inside the array is the CopyPlugin.

Any help is welcome.

My current code
```javascript
'use strict';

const CopyPlugin = require('copy-webpack-plugin');

module.exports = {
    options: {
        buildType: 'spa',
    },
    modifyWebpackConfig(opts) {
        const config = opts.webpackConfig;

        const pluginArray = config.plugins;

        return config;
    }
};
```

**Comment by fivethreeo** [comment, marked as answer]
```js
      const copyIndex = newWebpackConfig.plugins.findIndex((item) => item.constructor.name === 'CopyPlugin');
      newWebpackConfig.plugins.splice(copyIndex, 1);
```
